backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/services/ViewNodeService.java

```diff
@@ -289,6 +291,18 @@ public boolean showAnnotatingNode(Element element, IDiagramContext diagramContex
         return displayAnnotatingNode;
     }
 
+    public boolean isInFeature(Feature feature) {
```

Comment:
These services should be put in another service class as they have nothing to do with views or nodes.
May be in `UtilService`?